### PR TITLE
feat: Multi Dev Portal

### PR DESCRIPTION
**Pull request by cloudjumpercat**
### Description

 This PR documents the new multi-portal functionality and also does some docs cleanup in the Dev Portal section to make the user flow a bit clearer.

DOCU-3864
### Testing instructions

Preview links:
- New page: https://deploy-preview-7561--kongdocs.netlify.app/konnect/dev-portal/create-dev-portal/
- Revised:
  - https://deploy-preview-7561--kongdocs.netlify.app/konnect/api-products/productize-service/
  - https://deploy-preview-7561--kongdocs.netlify.app/konnect/dev-portal/applications/enable-app-reg/
  - https://deploy-preview-7561--kongdocs.netlify.app/konnect/dev-portal/
  - https://deploy-preview-7561--kongdocs.netlify.app/konnect/dev-portal/publish-service/

### Checklist 

- [x] Review label added 
- [x] [Conditional version tags](https://docs.konghq.com/contributing/conditional-rendering/#conditionally-render-content-by-version) added, if applicable.

### CODE REVIEW

**Comment by cloudjumpercat** [comment]
Hi @c3pko ! I finished drafting the multi-portal doc changes and was wondering if you could look them over and let me know if I missed anything/anything is incorrect:

New page: https://deploy-preview-7561--kongdocs.netlify.app/konnect/dev-portal/create-dev-portal/

Revised:
- https://deploy-preview-7561--kongdocs.netlify.app/konnect/api-products/productize-service/
- https://deploy-preview-7561--kongdocs.netlify.app/konnect/dev-portal/applications/enable-app-reg/
- https://deploy-preview-7561--kongdocs.netlify.app/konnect/dev-portal/
- https://deploy-preview-7561--kongdocs.netlify.app/konnect/dev-portal/publish-service/

 
I had a question about the API updates you mentioned [here](https://kongstrong.slack.com/archives/C045Y8G7VDW/p1718220724878109?thread_ts=1718220709.944269&cid=C045Y8G7VDW). I didn’t see them in the [platform-api repo](https://github.com/Kong/platform-api/pulls), so I just wanted to see if those were in the queue to be updated there when this feature goes GA.

File: app/konnect/api-products/productize-service.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ The response body will include an `id` field, denoting the unique identifier for
 {% navtab Konnect UI %}
 After creating a new API product, you can attach an API product version to it.
 
-1. From the API product builder, select **Product Versions**, then select **New Version**.
+1. In {% konnect_icon api-product %} [**API Products**](https://cloud.konghq.com/api-products), click the API product you want to create the version for and then click **Product Versions**, then click **New Version**.
 
 1. Enter a version name. For example `v1`.
      A version name can be any string containing letters, numbers, or characters;
```
Comment (c3pko) [inline app/konnect/api-products/productize-service.md:53]: change "A service can have multiple versions." to "An API Product can have multiple versions."

File: app/konnect/api-products/productize-service.md [hunk 2]
```diff
@@ -102,19 +102,38 @@ After creating the new version, you will see **Link with a Gateway Service** as
 {% navtabs %}
 {% navtab Konnect UI %}
 
-API products can be published to the Dev Portal. To publish your new API product navigate to the {% konnect_icon api-product %} [**API Products**](https://cloud.konghq.com/api-products) and follow these steps: 
+1. In {% konnect_icon api-product %} [**API Products**](https://cloud.konghq.com/api-products), select the API product that you created in the previous step.
+1. Click **Add** next to the API product or products you want to publish to the Dev Portal and then click **Finish**.
```
Comment (c3pko) [inline app/konnect/api-products/productize-service.md:106]:
```suggestion
1. Click **Add** on the API Product Overview page and select "Publish to Dev Portals" from the drop down menu. You will see a modal to prompting you to select which Dev Portals you want to publish your API Product to. Click "Publish" for the Dev Portals you want to publish it. Then click **Finish**. 
```

File: app/konnect/api-products/productize-service.md [hunk 3]
```diff
@@ -102,19 +102,38 @@ After creating the new version, you will see **Link with a Gateway Service** as
 {% navtabs %}
 {% navtab Konnect UI %}
 
-API products can be published to the Dev Portal. To publish your new API product navigate to the {% konnect_icon api-product %} [**API Products**](https://cloud.konghq.com/api-products) and follow these steps: 
+1. In {% konnect_icon api-product %} [**API Products**](https://cloud.konghq.com/api-products), select the API product that you created in the previous step.
+1. Click **Add** next to the API product or products you want to publish to the Dev Portal and then click **Finish**. 
+1. Repeat steps 1-3 for any other Dev Portals you want to add the API product to.
```
Comment (c3pko) [inline app/konnect/api-products/productize-service.md:107]:
```suggestion
```

File: app/konnect/api-products/productize-service.md [hunk 4]
```diff
@@ -102,19 +102,38 @@ After creating the new version, you will see **Link with a Gateway Service** as
...
+1. In {% konnect_icon api-product %} [**API Products**](https://cloud.konghq.com/api-products), select the API product you added to the Dev Portal. 
+1. Click **Product Versions** in the sidebar.
+1. Click the product version you created previously and in the **Dev Portals** tab, click **Browse Dev Portals** and select the Dev Portals you want to add the product version to.
```
Comment (c3pko) [inline app/konnect/api-products/productize-service.md:110]:
```suggestion
1. Click the product version you created previously and either under the **Actions** button or the **Dev Portals** tab, click **Publish to Dev Portals** and select the Dev Portals you want to add the product version to.

If want to require authentication on your API Product version, toggle **Require Authentication** to true. Note: If you require authentication you will need to select which authentication strategy your API should use. Portal admins can set default auth strategies for their portals so that API Owners don't need to worry about which auth strategy to select. By default, every portal comes with a key auth auth strategy.
```

File: app/konnect/api-products/productize-service.md [hunk 5]
```diff
@@ -102,19 +102,38 @@ After creating the new version, you will see **Link with a Gateway Service** as
...
 {% navtab Konnect UI %}
 
-API products can be published to the Dev Portal. To publish your new API product navigate to the {% konnect_icon api-product %} [**API Products**](https://cloud.konghq.com/api-products) and follow these steps: 
+1. In {% konnect_icon api-product %} [**API Products**](https://cloud.konghq.com/api-products), select the API product that you created in the previous step.
+1. Click **Add** next to the API product or products you want to publish to the Dev Portal and then click **Finish**. 
+1. Repeat steps 1-3 for any other Dev Portals you want to add the API product to.
+1. In {% konnect_icon api-product %} [**API Products**](https://cloud.konghq.com/api-products), select the API product you added to the Dev Portal. 
+1. Click **Product Versions** in the sidebar.
+1. Click the product version you created previously and in the **Dev Portals** tab, click **Browse Dev Portals** and select the Dev Portals you want to add the product version to. 
+    Product versions are published by default when you select Dev Portals, you can disable publishing while configuring the Dev Portals if you'd like to publish them later.
```
Comment (c3pko) [inline app/konnect/api-products/productize-service.md:111]:
```suggestion

```

File: app/konnect/api-products/productize-service.md [hunk 6]
```diff
@@ -20,7 +20,7 @@ You can set up an API product and API product version by clicking {% konnect_ico
 
 1. Select **API Product** from the API products dashboard to add a new API product.
 
-1. Create a new name for your API product, and enter an optional **Description** and any **labels** that you want to associate with the product, then press **create**. 
+1. Create a new name for your API product, and enter an optional **Description** and any **labels** that you want to associate with the product, then click **Create**. 
 
 You will be greeted by the dashboard for the API product that you just created. You can use this dashboard to manage an API product. You can read more about this dashboard on the API products [overview page](/konnect/api-products/)
 {% endnavtab %}
```
Comment (c3pko) [inline app/konnect/api-products/productize-service.md:26]:
```suggestion
{% endnavtab %}

can we format lines 31-40? the curl command is not formatted correctly: https://deploy-preview-7561--kongdocs.netlify.app/konnect/api-products/productize-service/#create-an-api-product
```

File: app/konnect/api-products/productize-service.md [hunk 7]
```diff
@@ -20,7 +20,7 @@ You can set up an API product and API product version by clicking {% konnect_ico
 
 1. Select **API Product** from the API products dashboard to add a new API product.
 
-1. Create a new name for your API product, and enter an optional **Description** and any **labels** that you want to associate with the product, then press **create**. 
+1. Create a new name for your API product, and enter an optional **Description** and any **labels** that you want to associate with the product, then click **Create**.
```
Comment (c3pko) [inline app/konnect/api-products/productize-service.md:24]: can we format lines 31-40? the curl command is not formatted correctly: https://deploy-preview-7561--kongdocs.netlify.app/konnect/api-products/productize-service/#create-an-api-product

File: app/konnect/api-products/productize-service.md [hunk 8]
```diff
@@ -102,19 +102,38 @@ After creating the new version, you will see **Link with a Gateway Service** as
...
+        --url https://{region}.api.konghq.com/v2/api-products/{ApiProductId} \
+        --header 'Authorization: <personal-access-token>' \
+        --header 'Content-Type: application/json' \
+        --data '{
+        "portal_ids": [
+            "4602b8cc-92fb-4197-b836-3eab20b16147"
+        ]
+    }'
+    ```
+    Be sure to replace `{ApiProductId}`, `name`, and `portal_ids` with your own values.
```
Comment (c3pko) [inline app/konnect/api-products/productize-service.md:130]: where is 'name'? I don't see that used anywhere

File: app/konnect/api-products/productize-service.md [hunk 9]
```diff
@@ -102,19 +102,38 @@ After creating the new version, you will see **Link with a Gateway Service** as
...
+The API product and product versions should now display in the Dev Portals you selected.
 {% endnavtab %}
 {% navtab API %}
-You can publish an API product by issuing a `POST` request to the [`/api-product-versions/`](/konnect/api/api-products/latest/#/API%20Product%20Versions/create-api-product-version) endpoint. Ensure you replace `{ApiProductId}` with the API product ID returned in the previous step.. 
+
+1. Before you publish the API product version, you must first assign the API product to any Dev Portals by issuing a `PATCH` request to the [`/api-products/{id}](/konnect/api/api-products/latest/#/API%20Products/update-api-product) endpoint:
+
+    ```sh
+    curl --request PATCH \
+        --url https://{region}.api.konghq.com/v2/api-products/{ApiProductId} \
```
Comment (c3pko) [inline app/konnect/api-products/productize-service.md:121]: 'ApiProductId' should just be 'id' right? https://docs.konghq.com/konnect/api/api-products/latest/#/API%20Products/update-api-product

File: app/konnect/api-products/productize-service.md [hunk 10]
```diff
@@ -102,19 +102,38 @@ After creating the new version, you will see **Link with a Gateway Service** as
...
+1. In {% konnect_icon api-product %} [**API Products**](https://cloud.konghq.com/api-products), select the API product you added to the Dev Portal. 
+1. Click **Product Versions** in the sidebar.
+1. Click the product version you created previously and in the **Dev Portals** tab, click **Browse Dev Portals** and select the Dev Portals you want to add the product version to. 
+    Product versions are published by default when you select Dev Portals, you can disable publishing while configuring the Dev Portals if you'd like to publish them later.
 
-1. Select the API product that you created in the previous step.
-2. Navigate to the **Actions** menu, then select **publish**. 
-
-Your API product is now consumable by developers from the **Dev Portal**.
+The API product and product versions should now display in the Dev Portals you selected.
```
Comment (c3pko) [inline app/konnect/api-products/productize-service.md:113]:
```suggestion
The API product and product versions you published should now be displayed in the Dev Portals you selected.
```

File: app/konnect/api-products/productize-service.md [hunk 11]
```diff
@@ -102,19 +102,38 @@ After creating the new version, you will see **Link with a Gateway Service** as
...
+
+1. You can publish an API product version by issuing a `POST` request to the [`/api-product-versions/`](/konnect/api/api-products/latest/#/API%20Product%20Versions/create-api-product-version) endpoint. Ensure you replace `{ApiProductId}` with the API product ID returned in the previous step. 
 
     ```sh
     curl -X 'POST' \
-        'https://{region).api.konghq.com/v2/api-products/{ApiProductId}/product-versions' \
+        'https://{region}.api.konghq.com/v2/api-products/{ApiProductId}/product-versions' \
         -H 'accept: application/json' \
         -H 'Authorization: Bearer <personal-access-token>' \
         -H 'Content-Type: application/json' \
```
Comment (c3pko) [inline app/konnect/api-products/productize-service.md:139]: Can we update the summary (lines 148-154) to:

## Summary
In this section, you created an API product and created an API Product version with a linked Gateway Service. You then published both the API Product and API Product version to one or more Dev Portals.

## More information
* [Add API product documentation](/konnect/dev-portal/publish-service/): Learn how to add your API spec and API product documentation to the Dev Portal so your developers can more easily consume your APIs.

File: app/konnect/dev-portal/access-and-approval/manage-app-reg-requests.md [hunk 12]
```diff
@@ -71,4 +71,19 @@ The following status conditions reflect the current state of a registration requ
...
+Each application has its own dashboard, which provides a high-level summary of the **Number of Requests**, **Average Error Rate**, and **p99 Latency**, and charts for the following data points: 
+
+* Requests by version
+* P99 latency by version
+* Error code distribution 
+
+All of these metrics can be viewed within a selected time frame of up to **Last 30 days**, over a period of the last 90 days.
+
+{:.note}
+> Free accounts only have access to a time frame of 24 hours.
```
Comment (c3pko) [inline app/konnect/dev-portal/access-and-approval/manage-app-reg-requests.md:89]: This is not really related to app reg so I would suggest moving it to a different page. Perhaps https://docs.konghq.com/konnect/dev-portal/applications/application-overview/ is a more appropriate location for it

File: app/konnect/dev-portal/access-and-approval/manage-app-reg-requests.md [hunk 13]
```diff
@@ -71,4 +71,19 @@ The following status conditions reflect the current state of a registration requ
...
 **Rejected**
-: An application registration request was rejected by a {{site.konnect_short_name}} admin.
\ No newline at end of file
+: An application registration request was rejected by a {{site.konnect_short_name}} admin.
+
+## Contextual developer application analytics 
+
+You can view analytics for developer applications from within the {{site.konnect_short_name}} Dev Portal. This gives developers the ability to gain insights into all of their applications and the relationships between applications and APIs.
+
+Each application has its own dashboard, which provides a high-level summary of the **Number of Requests**, **Average Error Rate**, and **p99 Latency**, and charts for the following data points:
```
Comment (c3pko) [inline app/konnect/dev-portal/access-and-approval/manage-app-reg-requests.md:80]:
```suggestion
You can view analytics for your applications from within the {{site.konnect_short_name}} Dev Portal. This helps you gain insight into your consumption of different API versions, routes, and methods.

Each application has its own dashboard, which provides a high-level summary of the **Number of Requests**, **Average Error Rate**, and **p99 Latency**, and charts for the following data points: 
```

File: app/konnect/dev-portal/applications/enable-app-reg.md [hunk 14]
```diff
@@ -43,46 +43,39 @@ For non-`default` control planes, app registration is supported using the `konne
...
+- [An API product that is versioned and published to a {{site.konnect_short_name}} Dev Portal](/konnect/dev-portal/publish-service) so that it appears in the catalog.
 
 - If you are using [OpenID Connect](#oidc-flow) for your authorization:
 
   - Set up your application, claims, and scopes in your OpenID identity provider. Refer to your IdP/OP provider's documentation for instructions.
 
   - Edit the **Reference ID** field in the Dev Portal
     [Update Application](/konnect/dev-portal/applications/dev-apps#edit-my-app)
-    dialog to match to your third-party OAuth2 claim.
+    dialog to match to your third-party OAuth2 claim. <!-- is this on the admin side of Dev Portal UI, or dev side?-->
```
Comment (c3pko) [inline app/konnect/dev-portal/applications/enable-app-reg.md:54]: this is on the dev portal side

File: app/konnect/dev-portal/applications/enable-app-reg.md [hunk 15]
```diff
@@ -43,46 +43,39 @@ For non-`default` control planes, app registration is supported using the `konne
...
+1. Click **Product Versions** and select a product version.
 
-4. Toggle the **Application registration enabled** button to true to allow Dev Portal developers to register their applications for the API product version.
+  {:.note}
+  > **Note:** If the API product version is in the `default` control plane group, it will
+  instead receive read-only entries for the `acl` and `key-auth` plugins to provide
+  support for {{site.base_gateway}} versions less than 3.0.
 
-5. Optional: click to enable [**Auto Approve**](/konnect/dev-portal/access-and-approval/auto-approve-devs-apps/) for application registration requests
+1. From the **Dev Portals** tab, click **Publish API**.
```
Comment (c3pko) [inline app/konnect/dev-portal/applications/enable-app-reg.md:70]:
```suggestion
1. From the **Dev Portals** tab, click **Publish to Dev Portals**.
```

File: app/konnect/dev-portal/applications/enable-app-reg.md [hunk 16]
```diff
@@ -43,46 +43,39 @@ For non-`default` control planes, app registration is supported using the `konne
...
 
-6. Click save to enable application registration and apply the auth strategy for that product version
+1. Enable **Require Authentication** and select **key-auth** for your auth strategy for each Dev Portal.
```
Comment (c3pko) [inline app/konnect/dev-portal/applications/enable-app-reg.md:72]:
```suggestion
1. Enable authentication by toggling on **Require Authentication**. If your API Product is already published to a Dev Portal, select **key-auth** in the Auth Strategy drop down for that Dev Portal. If your API Product is not yet published click **+ Dev Portal**, select the target Dev Portal in the dropdown, and then select **Key Auth** in the Auth Strategy dropdown selection.
```

File: app/konnect/dev-portal/applications/enable-app-reg.md [hunk 17]
```diff
@@ -43,46 +43,39 @@ For non-`default` control planes, app registration is supported using the `konne
...
 
-7. Click **Status** and update it to **Published** to publish the product version to the Dev Portal product catalog
+1. Enable **Accept application registrations** for each Dev Portal.
```
Comment (c3pko) [inline app/konnect/dev-portal/applications/enable-app-reg.md:74]:
```suggestion
1. Enable **Accept application registrations** for each Dev Portal to allow developers to register their applications to consume this API.
```

File: app/konnect/dev-portal/applications/enable-app-reg.md [hunk 18]
```diff
@@ -43,46 +43,39 @@ For non-`default` control planes, app registration is supported using the `konne
...
-    read-only entry for the `konnect-application-auth` plugin.
+5. Optional: Click the [**Auto Approve**](/konnect/dev-portal/access-and-approval/auto-approve-devs-apps/) checkbox if you want to automatically approve all application registration requests.
 
-{:.note}
-> **Note:** If the API product version is in the `default` control plane group, it will
-instead receive read-only entries for the `acl` and `key-auth` plugins to provide
-support for {{site.base_gateway}} versions less than 3.0.
+1. Click **Save**.
 
 ## Enable app registration with OpenID Connect {#oidc-flow}
```
Comment (c3pko) [inline app/konnect/dev-portal/applications/enable-app-reg.md:80]: need to update the steps for oidc too

File: app/konnect/dev-portal/create-dev-portal.md [hunk 19]
```diff
@@ -0,0 +1,156 @@
+---
+title: Create a Dev Portal
+---
+
+The {{site.konnect_short_name}} Dev Portal is a customizable website for developers to locate, access, and consume API services. You can create both public and private Dev Portals, depending on your use case. With a public Dev Portal, anyone with the URL can view the APIs you add to your Dev Portal. With a private Dev Portal, only those who have signed in to the Dev Portal can see the APIs you add to your Dev Portal. Additionally, when you configure private Dev Portals, you can also allow developers to register apps with your APIs. You can also publish your APIs to both a private Dev Portal as well as a public one, such as if you want to have one Dev Portal for internal developers at your company as well as allow external developers at other companies to create apps with some of your APIs.
```
Comment (c3pko) [inline app/konnect/dev-portal/create-dev-portal.md:5]:
```suggestion
The {{site.konnect_short_name}} Dev Portal is a customizable website for developers to locate, access, and consume API services.

Users can create both public and private Dev Portals, depending on their use case. Public Dev Portals do not require users to login to see published APIs, and are discoverable on the internet. Private Dev Portals require users to create an account and login in order to see published APIs. Coming soon, Konnect will support Dev Portals that contain pages that are public _and_ private, allowing the public to browse the publicly available catalog, but requiring users to login to see a full catalog, and begin consuming those APIs.

When Portal admins require developers to create an account and login to use the Dev Portal, developers can "self serve" their API consumption. Developers can create applications, register them to their target APIs, and generate credentials to start consuming those APIs.
```

File: app/konnect/dev-portal/create-dev-portal.md [hunk 20]
```diff
@@ -0,0 +1,156 @@
...
+
+## Prerequisites
+
+Configure an authentication strategy if you plan to use one. This auth strategy will be how developers authenticate when they use your APIs. <!-- tried searching for a link to a doc for this, is it just setting up an auth plugin?-->
```
Comment (c3pko) [inline app/konnect/dev-portal/create-dev-portal.md:9]:
```suggestion
Configure an [authentication strategy](https://deploy-preview-7561--kongdocs.netlify.app/konnect/dev-portal/applications/enable-app-reg/) if you plan to use one. This auth strategy will be how developers authenticate when they use your APIs. <!-- tried searching for a link to a doc for this, is it just setting up an auth plugin?-->
```

File: app/konnect/dev-portal/create-dev-portal.md [hunk 21]
```diff
@@ -0,0 +1,156 @@
...
+
+## Steps
+{% navtabs %}
+{% navtab Konnect UI %}
+1. In {% konnect_icon dev-portal %} [**Dev Portal**](https://cloud.konghq.com/portal), click **Dev Portal** on the top right.
+1. In the Create Dev Portal dialog, configure your Dev Portal settings:
+    1. Enter general information for your Dev Portal, including a name and optional description.
+    1. Select if you want the Dev Portal to be private or public. Developers can only register apps to private Dev Portals.
```
Comment (c3pko) [inline app/konnect/dev-portal/create-dev-portal.md:17]:
```suggestion
    1. Select if you want the Dev Portal to be private or public. Anyone can see APIs published to a public portal, only registered and logged in users can see APIs in a private portal. Note: Developers can only get credentials and start consuming APIs in a self serve way in a private Dev Portal.
```

File: app/konnect/dev-portal/create-dev-portal.md [hunk 22]
```diff
@@ -0,0 +1,156 @@
...
+    1. Private Dev Portal only: Configure the following settings:
+        * **Auto Approve Developers**: If you enable this setting, this will allow developers to automatically be approved when they sign up for your private Dev Portal. If you don't enable this, you must configure single sign-on or manually approve developers as they register. 
+        * **Auto Approve Applications**: If you enable this setting, apps that developers create with your APIs will be automatically approved. If you don't enable this, you must manually approve or deny application registrations.
+        * **Portal RBAC**: You must enable this setting if you plan to allow developers to register with your Dev Portal. Portal RBAC allows you to assign developers to teams and roles that determine if they can only view or view and consume the APIs in your Dev Portal.
```
Comment (c3pko) [inline app/konnect/dev-portal/create-dev-portal.md:21]:
```suggestion
        * **Portal RBAC**: Portal RBAC allows you to assign developers to teams and roles that determine if they can only view or view and consume the APIs in your Dev Portal.
```

Comment (c3pko) [reply app/konnect/dev-portal/create-dev-portal.md:21]: this "You must enable this setting if you plan to allow developers to register with your Dev Portal. " is not true

File: app/konnect/dev-portal/create-dev-portal.md [hunk 23]
```diff
@@ -0,0 +1,156 @@
...
+        * **Default Application Auth Strategy**: Select the auth strategy you want the APIs in your Dev Portal to use if they require authentication. This auth strategy will be how developers authenticate when they use your APIs.
```
Comment (c3pko) [inline app/konnect/dev-portal/create-dev-portal.md:22]:
```suggestion
        * **Default Application Auth Strategy**: Select the auth strategy you want the APIs in your Dev Portal to use if they require authentication. This auth strategy will be how developers authenticate when they register to consume your APIs. Note: Selecting a default auth strategy makes it easier for your API Product Owners to publish an API to a Dev Portal that requires authentication, as the default auth strategy will be auto selected for them. If you're not sure which auth strategy to use you can select the pre-created key-auth strategy.
```

File: app/konnect/dev-portal/create-dev-portal.md [hunk 24]
```diff
@@ -0,0 +1,156 @@
...
+
+{% endnavtab %}
+{% endnavtabs %}
+
+## Access the Dev Portal
+
+You can access the Dev Portal using the Dev Portal URL. The URL is displayed when you click {% konnect_icon dev-portal %} **Dev Portal** to open the **Published API Products** overview page.
+The Dev Portal URL looks like this: 
+    
+    https://example.us.portal.konghq.com/
```
Comment (c3pko) [inline app/konnect/dev-portal/create-dev-portal.md:135]:
```suggestion
    https://{portalId}.{region}.portal.konghq.com/
```

File: app/konnect/dev-portal/index.md [hunk 25]
```diff
@@ -5,24 +5,17 @@ content_type: explanation
...
-You can have different API specs that correspond to different API product versions.
-{{site.konnect_short_name}} accepts OpenAPI (Swagger) specs in YAML or JSON.
-
-Once you've uploaded the spec, you can also preview the way the spec will render, including the methods available, endpoint descriptions, and example values. You'll also be able to filter by tag when in full-page view. 
-The Dev Portal lives at a separate URL from {{site.konnect_short_name}} and requires that all users, including Admin roles, [register as a Developer](/konnect/dev-portal/dev-reg/).
+| You want to... | Then use... |
+| -------------- | ----------- |
+| Publish APIs to two Dev Portals; one for internal developers and one for external partner developers | [Multi-portal](/konnect/dev-portal/create-dev-portal/) |
+| Let developers get API keys and start using your APIs | [Enable app reg and Dev Portal](/konnect/dev-portal/applications/enable-app-reg/) |
+| Segment which users can see which APIs | [Assign different APIs and permissions with RBAC Teams](/konnect/api/portal-auth/portal-rbac-guide/#main) |
```
Comment (c3pko) [inline app/konnect/dev-portal/index.md:16]: added comments in a pr elsewhere...struggling to find it but I think we should revert this page to how it was before

File: app/konnect/dev-portal/publish-service.md [hunk 26]
```diff
@@ -2,12 +2,14 @@
 title: Add and publish API product documentation
 ---
 
-The Dev Portal is an API catalog that lets you document your {{site.konnect_short_name}} API products
+The Dev Portal is an API catalog that lets you document and publish your {{site.konnect_short_name}} API products
 and share them with your developers. Developers can use the Dev Portal to
-locate, access, consume, and register applications against the products.
+locate, access, consume, and register applications against the products.
```
Comment (c3pko) [inline app/konnect/dev-portal/publish-service.md:7]:
```suggestion
locate, access, consume, and register applications to the products. 
```

File: app/konnect/updates.md [hunk 27]
```diff
@@ -9,6 +9,11 @@ an application that lets you manage configuration for multiple runtimes
 from a single, cloud-based control plane, and provides a catalog of all deployed
 services. [Try it today!](https://cloud.konghq.com/quick-start)
 
+## July 2024
+
+**Multi-portal**
+: {{site.konnect_short_name}} now supports multiple Dev Portals. This allows you to better segment your customers, brands, and API Product experiences. For more information, see [Create Dev Portal](/konnect/dev-portal/create-dev-portal)
```
Comment (c3pko) [inline app/konnect/updates.md:15]:
```suggestion
: {{site.konnect_short_name}} now supports multiple Dev Portals. Users can create multiple Dev Portals with the same great functionality in order to better segment their customers, brands, and API Product experiences. For more information, see [Create Dev Portal](/konnect/dev-portal/create-dev-portal)
```

File: app/konnect/dev-portal/publish-service.md [hunk 28]
```diff
@@ -61,14 +63,20 @@ use the [sample Analytics spec](/konnect/vitalsSpec.yaml) for testing.
 This OpenAPI spec will be shown under the version name when this service is
 published to the Dev Portal.
 
-## Publish API Product to the Dev Portal
+## Publish an API product to a Dev Portal
```
Comment (c3pko) [inline app/konnect/dev-portal/publish-service.md:66]:
```suggestion
## Publish an API product and API Product version to a Dev Portal
```

File: app/konnect/dev-portal/publish-service.md [hunk 29]
```diff
@@ -61,14 +63,20 @@ use the [sample Analytics spec](/konnect/vitalsSpec.yaml) for testing.
...
+
+To publish an API, you must first add the API product to a Dev Portal and then add an API product version.
```
Comment (c3pko) [inline app/konnect/dev-portal/publish-service.md:68]:
```suggestion
To publish an API, you must publish both the API Product and API Product version to the Dev Portal.
```

File: app/konnect/dev-portal/publish-service.md [hunk 30]
```diff
@@ -61,14 +63,20 @@ use the [sample Analytics spec](/konnect/vitalsSpec.yaml) for testing.
...
 
-To view an API Product's documentation alongside its dynamic API reference (based on the API specification), 
-navigate to {% konnect_icon api-product %} [**API Products**](https://cloud.konghq.com/api-products) and publish your API product along with its API version.
+1. In {% konnect_icon dev-portal %} [**Dev Portal**](https://cloud.konghq.com/portal), select the Dev Portal you want to publish the API for.
+1. Click **API Products**.
```
Comment (c3pko) [inline app/konnect/dev-portal/publish-service.md:71]:
```suggestion
1. Click **[API Products](https://cloud.konghq.com/us/api-products/)**.
```

File: app/konnect/dev-portal/publish-service.md [hunk 31]
```diff
@@ -61,14 +63,20 @@ use the [sample Analytics spec](/konnect/vitalsSpec.yaml) for testing.
...
+1. Click **Add** next to the API product or products you want to publish to the Dev Portal and then click **Finish**.
```
Comment (c3pko) [inline app/konnect/dev-portal/publish-service.md:72]:
```suggestion
+ 1. Select the API Product you want to publish. On the API Product overview page click **Add** and select "Publish to Dev Portals" in the dropdown menu.
+1. Click "Publish" to publish your API Product to a specific Dev Portal 
```

File: app/konnect/dev-portal/publish-service.md [hunk 32]
```diff
@@ -61,14 +63,20 @@ use the [sample Analytics spec](/konnect/vitalsSpec.yaml) for testing.
...
+## Publish an API product to a Dev Portal
+
+To publish an API, you must first add the API product to a Dev Portal and then add an API product version.
 
-To view an API Product's documentation alongside its dynamic API reference (based on the API specification), 
-navigate to {% konnect_icon api-product %} [**API Products**](https://cloud.konghq.com/api-products) and publish your API product along with its API version.
+1. In {% konnect_icon dev-portal %} [**Dev Portal**](https://cloud.konghq.com/portal), select the Dev Portal you want to publish the API for.
+1. Click **API Products**.
+1. Click **Add** next to the API product or products you want to publish to the Dev Portal and then click **Finish**. 
+1. Repeat steps 1-3 for any other Dev Portals you want to add the API product to.
```
Comment (c3pko) [inline app/konnect/dev-portal/publish-service.md:73]:
```suggestion
```

File: app/konnect/dev-portal/publish-service.md [hunk 33]
```diff
@@ -61,14 +63,20 @@ use the [sample Analytics spec](/konnect/vitalsSpec.yaml) for testing.
...
+1. In {% konnect_icon api-product %} [**API Products**](https://cloud.konghq.com/api-products), select the API product you added to the Dev Portal. 
+1. Click **Product Versions** in the sidebar.
+1. Click the product version you created previously and in the **Dev Portals** tab, click **Browse Dev Portals** and select the Dev Portals you want to add the product version to.
```
Comment (c3pko) [inline app/konnect/dev-portal/publish-service.md:76]:
```suggestion
1. Click the product version you created previously and want to publish to your Dev Portal(s). From the **Actions** dropdown menu or the **Dev Portals** tab, click **Publish to Dev Portals** and select the Dev Portals you want to publish the product version to. 
```

File: app/konnect/dev-portal/applications/enable-app-reg.md [hunk 34]
```diff
@@ -43,46 +43,39 @@ For non-`default` control planes, app registration is supported using the `konne
...
-- An API product that is versioned and published to the
-  {{site.konnect_short_name}} Dev Portal so that it appears in the catalog.
+- [An API product that is versioned and published to a {{site.konnect_short_name}} Dev Portal](/konnect/dev-portal/publish-service) so that it appears in the catalog.
 
 - If you are using [OpenID Connect](#oidc-flow) for your authorization:
 
   - Set up your application, claims, and scopes in your OpenID identity provider. Refer to your IdP/OP provider's documentation for instructions.
 
   - Edit the **Reference ID** field in the Dev Portal
     [Update Application](/konnect/dev-portal/applications/dev-apps#edit-my-app)
```
Comment (c3pko) [inline app/konnect/dev-portal/applications/enable-app-reg.md:53]: this step is from the dev side, which doesn't make sense because the rest of the instructions (and the page) are from the admin side. I suggest removing this sentence
```suggestion
```

**Review by c3pko — COMMENTED** [review]
one more change

File: app/konnect/dev-portal/index.md
Comment (cloudjumpercat) [inline app/konnect/dev-portal/index.md:1]: For writer review: I moved a lot of the content out of here to the relevant pages because I originally added a diagram. Ella gave the feedback that the diagram wasn't quite accurate and wouldn't be ready by GA, so we removed it. Does this page work with the info that's currently on it, or does it need the feature sections back?

File: app/konnect/dev-portal/applications/enable-app-reg.md [hunk 35]
```diff
@@ -43,46 +43,35 @@ For non-`default` control planes, app registration is supported using the `konne
```
[the remark this answers is not in the document]
Comment (cloudjumpercat) [reply app/konnect/dev-portal/applications/enable-app-reg.md:43]: @Guaris I made https://konghq.atlassian.net/browse/DOCU-3951 to handle the couple misc feedback you had here

**Comment by acgoldsmith** [comment]
@cloudjumpercat - what do you think about using "authentication strategy" rather than "auth strategy", except when referring to UI use of that term?

**Comment by acgoldsmith** [comment]
@cloudjumpercat , I *really* like the new devportal overview page! The use case table and audience tabs are a great approach.

**Comment by acgoldsmith** [comment]
LGTM!!! Approving now, I woudn't gate release on swapping out the "auth strategy" for proper term.

**Comment by cloudjumpercat** [comment]
@acgoldsmith Thanks for the review! It looks like you maybe commented for your review instead of approving. I added your suggestion about changing "auth strategy" to the multi-portal fast follow ticket here: https://konghq.atlassian.net/jira/software/c/projects/DOCU/boards/58?selectedIssue=DOCU-3951

File: app/konnect/dev-portal/create-dev-portal.md [hunk 36]
```diff
@@ -0,0 +1,115 @@
...
+    ```
+    Be sure to replace the PAT as well as the following placeholders with your own values: 
+    * `name`
+    * `rbac_enabled`
+    * `auto_approve_applications`
+    * `auto_approve_developers`
+    * `custom_domain`
+    * `custom_client_domain`
+    * `default_application_auth_strategy_id`
+    Refer to the API spec for more information.
```
Comment (cloudjumpercat) [inline app/konnect/dev-portal/create-dev-portal.md:63]:
```suggestion
    * `default_application_auth_strategy_id`
    
    Refer to the API spec for more information. 
```

**Review by lena-larionova — APPROVED** [review]

**Comment by cloudjumpercat** [comment]
Vale failure is expected as there was a Vale update
